css/index.css

```
@@ -1,3 +1,456 @@
+}
+
+.copyright {
+	text-align: center;
+	margin-top: 40px;
+}
+
+.copyright-larger {
+	font-size: 100%;
+	margin-top: 100px;
+	text-align: center;
+}
+
+address p {
+	/* line-height: 1.5; */
+	margin-bottom: 10px;
+}
+
+.services-header-img {
+	margin-top: 30px;
+	margin-bottom: 30px;
+}
+
+ul {
+	list-style-type: square;
+	list-style-position: inside;
+	line-height: 2;
+	margin-top: 20px;
+}
+
+.services {
+	margin-bottom: -10px;
+}
+
+.services h1 {
+	font-size: 25px;
+}
+
+.boxes {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+}
+
+.boxes > section {
+	background-color: #F4F3F4;
+	border: 1px solid black;
+	width: 48%;
+	padding: 20px;
+	margin-bottom: 40px;
+}
+
+.boxes span {
+	border: 1px solid black;
+	width: 140px;
+	height: 35px;
+	border-radius: 5px;
+	background-color: white;
+	font-size: 80%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	margin-top: 20px;
+}
+
+.services-bottom {
+	display: flex;
+	justify-content: space-between;
+}
+
+.services-bottom div {
+	width: 45%;
+}
+
+.contact-header {
+	width: 100%;
+	margin-top: 20px;
+}
+
+.contact-header-text {
+	font-size: 40px;
+	margin-top: 20px;
+	margin-bottom: 10px;
+}
+
+.contact {
+	display: flex;
+	justify-content: space-between;
+}
+
+.contact-content {
+	width: 60%;
+}
+
+.typing {
+	width: 100%;
+	margin-top: 20px;
+}
+
+.contact-content > p {
+	margin-bottom: 10px;
+}
+
+.contact-content ul {
+	margin-top: -10px;
+	margin-left: 5px;
+}
+
+.content-form {
+	width: 40%;
+}
+
+.form {
+	list-style-type: none;
+	list-style-position: inside;
+	line-height: 2;
+	margin-top: 20px;
+	padding: 10px;
+	border: 1px solid black;
+	border-radius: 5px;
+	background-color: #F4F3F4;
+	
+}
+
+.contact-method {
+	list-style-type: none;
+	list-style-position: outside;
+}
+
+.contact-form input {
+	width: 100%;
+}
+
+.contact-form textarea {
+	width: 100%;
+}
+
+button {
+	border: 1px solid black;
+	width: 140px;
+	height: 35px;
+	border-radius: 5px;
+	background-color: white;
+	font-size: 80%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	margin-top: 20px;
+	margin-left: 20%;
+	margin-bottom: 10px;
+}
+
+.contact-content {
+	display: flex;
+}
+
+@media (max-width: 800px) {
+
+	.container {
+		width: 775px;
+		padding-left: 10px;
+	}
+
+	header {
+		flex-direction: column-reverse;
+	}
+	
+	nav {
+		padding-top: 35px;
+		padding-left: 10px;
+		width: 775px;
+		justify-content: space-around;
+	}
+
+	.code-image {
+		display: none;
+	}
+
+	.innovation {
+		justify-content: center;
+		width: 775px;
+		padding-left: 0;
+	}
+
+	.middle-img {
+		width: 760px;
+	}
+
+	.middle-content > div {
+		width: 360px;
+	}
+
+	.middle-content {
+		width: 740px;
+		padding-left: 10px;
+	}
+
+	.bottom-content {
+		width: 725px;
+		padding-left: 10px;
+	}
+
+	.bottom-content > div {
+		width: 215px;
+	}
+
+	hr {
+		width: 760px;
+	}
+
+	footer {
+		padding-left: 10px;
+	}
+	
+}
+
+@media (max-width: 500px) {
+	.container {
+		width: 500px;
```

The vertical scroll bar takes up about 20px so it's important to make your containers a bit smaller than the breakpoint to accommodate that. To get rid of the horizontal scroll bar that comes up, though, you'll want to use a percentage based width. This is probably how I would've done that:
```suggestion
		width: 98%;
		max-width: 480px;
```